examples/quirrel/app/routes/queues/greetings.ts

```diff
@@ -0,0 +1,3 @@
+import greetingsQueue from "~/queues/greetings.server";
+
+export const action = greetingsQueue;
```

Comment:
Is this code ever used?

Reply:
Yes, this is the action called by the Quirrel server. The actual body of the action is the second parameter passed to the `Quirrel()` constructor in `app/queues/greetings.server.ts`.

We can't directly export the action as below because Quirrel is supposed to be used only from the server and esbuild wouldn't be able to tree-shake the Quirrel import from the client-side bundle.
```ts
import { Queue } from "quirrel/remix";

export const action = Queue("/queues/greetings", async (name: string) => {
	console.log(`Greetings, ${name}!`);
});
```
```ts
import type { LoaderFunction } from "remix";

import { action as greetingsQueue } from "~/routes/queues/greetings";

export const loader: LoaderFunction = async () => {
	await greetingsQueue.enqueue("Groot"); // browser error

	return null;
};
```